python/ray/serve/backend_state.py

```diff
@@ -43,17 +40,21 @@ class BackendReplica:
     def __init__(self, controller_name: str, detached: bool,
                  replica_tag: ReplicaTag, backend_tag: BackendTag):
         self._actor_name = format_actor_name(replica_tag, controller_name)
+        self._placement_group_name = self._actor_name + "_placement_group"
         self._controller_name = controller_name
         self._detached = detached
         self._replica_tag = replica_tag
         self._backend_tag = backend_tag
         self._actor_handle = None
+        self._start_time = None
+        self._prev_slow_startup_warning_time = None
         self._startup_obj_ref = None
         self._drain_obj_ref = None
         self._state = ReplicaState.SHOULD_START
```

Comment:
Can you add `self._placement_group = None` to the constructor? To account for checkpointing before starting?

Reply:
yep good catch